src/test.props

```diff
@@ -8,5 +7,8 @@
 -->
   <PropertyGroup Condition=" $(MSBuildProjectName.EndsWith('.Tests')) AND '$(BuildingInsideVisualStudio)' == 'true' ">
     <OutputPath>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.proj))\\bin\\tests\\Windows_NT.$(Platform).$(Configuration)\$(MSBuildProjectName)\dnxcore50</OutputPath>
-  </PropertyGroup>  
+  </PropertyGroup>
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
+    <XunitOptions>-parallel none</XunitOptions>
```

Comment:
Yes would special case this for outerloop only for performance reasons. 
I would like to check this in an open a bug to enable parallel execution.